crates/bevy_ecs/src/event.rs

```diff
@@ -173,6 +176,15 @@ impl<'w, 's, T: Resource> EventWriter<'w, 's, T> {
     }
 }
 
+/// A simple event iterator which can be manually controlled.
+///
+/// In most cases, an [EventReader] will better meet your needs,
+/// as it both tracks which events a system has seen and fetches the relevant data.
+/// However, [EventReader] and [EventWriter] have conflicting accesses, and so cannot be used in the same system.
+///
+/// To get around this, you can store a [ManualEventReader] in a [Local] to help track events seen
+/// when working directly with `ResMut<Events<T>>`,
+/// allowing you to read and write events from within the same system.
```

Comment:
Should `ManualEventReader` be renamed given you can also "write" events with it?